lib/rev-api/models/order_request.rb

```diff
@@ -169,6 +194,50 @@ class Input < ApiSerializable
 
     # External URL, if sources wasn't POSTed as input (YouTube, Vimeo, Dropbox, etc)
     attr_reader :external_link
+
+    #Optional, list of glossary entries.
+    attr_reader :glossary
+
+    #Optional, list of speaker names.
+    attr_reader :speaker_names
+
+    # Optional, list of accents.
+    attr_reader :accents
+
+    SUPPORTED_ACCENTS = {
+      :american_neutral => 'AmericanNeutral',
+      :american_southern => 'AmericanSouthern',
+      :asian => 'Asian',
+      :australian => 'Australian',
+      :british => 'British',
+      :indian => 'Indian',
+      :other => 'Other',
+      :unknown => 'Unknown'
+    }
+
+    def validate_glossary
+      if !glossary.nil?
+        raise(ArgumentError, "Glossary must not exceed #{GLOSSARY_ENTRIES_LIMIT} entries") if glossary.length > GLOSSARY_ENTRIES_LIMIT
+        glossary.each { |term|
+          raise(ArgumentError, "Glossary entries cannot exceed #{GLOSSARY_ENTRY_LENGTH_LIMIT} characters") if term.length > GLOSSARY_ENTRY_LENGTH_LIMIT
```

Comment:
Here and everywhere. I don't think one-liners are good fit here since there is 140+ characters in the line. 
You can safely use `if ... end` which will be more readable.